### PR TITLE
ATS-140 : Release ATS 0.2-EA for ACS 6.1.0-EA3

### PR DESCRIPTION
**Pull request by tzclucian**
- updated versions

### CODE REVIEW

File: distribution/src/main/resources/VERSIONS.md [hunk 1]
```diff
@@ -12,8 +12,8 @@ This file lists the recommended components for this Service Pack. Use it along w
 | Alfresco Search Services | 1.2.0 | 1.1.1 |
 | Alfresco Insight Engine | 1.0.0 |
 | Alfresco Desktop Sync Service | @alfresco.desktop-sync.version@ |
-| Alfresco Transform Service | 0.1.1 EA |
-| Document Transformation Engine | @alfresco.transformation-engine.version@ |
```
Comment (montgolfiere) [inline distribution/src/main/resources/VERSIONS.md]: DTE is a separate product - please revert this line !!

Comment (tzclucian) [reply distribution/src/main/resources/VERSIONS.md]: I see engine, I update. Reverted 😃

**Comment by montgolfiere** [comment]
OK, although this is now pending the re-introduction of the image versions:

https://github.com/Alfresco/acs-packaging/commit/52a0fc0cc7e4586abfed3b660fd69e58d3eaa75b

cc: @alandavis

**Comment by montgolfiere** [comment]
FYI: superceded by https://github.com/Alfresco/acs-packaging/pull/71 (ATS 0.2.1 EA)

**Pull request closed without merging**